django/db/models/fields/__init__.py

```diff
@@ -126,6 +126,7 @@ class Field(RegisterLookupMixin):
     one_to_many = None
     one_to_one = None
     related_model = None
+    originating_model = None
```

Comment:
I think a comment here explaining the semantics of `originating_model` would be really useful.